src/hooks/tests/use-site-sync-management.test.ts

```diff
@@ -0,0 +1,142 @@
+import { renderHook, waitFor } from '@testing-library/react';
+import { useAuth } from '../use-auth';
+import { useFetchWpComSites } from '../use-fetch-wpcom-sites';
+import { useSiteDetails } from '../use-site-details';
+import { useSiteSyncManagement } from '../use-site-sync-management';
+
+jest.mock( '../use-auth' );
+jest.mock( '../use-site-details' );
+jest.mock( '../use-fetch-wpcom-sites' );
+
+export const mockConnectedWpcomSites = [
```

Comment:
What is the reason to export it?

Reply:
I think it was most a habit 😅 

Removed in https://github.com/Automattic/studio/pull/641/commits/a5566895b31504c7aec72b4a6abaf0335fce940c